app/views/structures/_cargocontainer.html.erb

```diff
@@ -38,7 +38,7 @@
         </table>
         
         <% structure = Structure.find(container_id) %>
-        <% if structure.user != current_user and structure.structure_type != 'wreck' and !structure.user.in_same_fleet_as(current_user.id) and structure.created_at > (DateTime.now.to_time - 10.minutes).to_datetime and !current_user.system.low? and !current_user.system.wormhole? %>
+        <% if structure.user != current_user and structure.structure_type != 'wreck' and !structure.user.in_same_fleet_as(current_user) and structure.created_at > (DateTime.now.to_time - 10.minutes).to_datetime and !current_user.system.low? and !current_user.system.wormhole? %>
```

Comment:
Can we also do this multiple lines ?

Reply:
lordy yes... that's crazy hard to read... but didn't want to change toooo much :)

Reply:
Haha you are good to go! 😄

Reply:
Updated.